src/TropicalGeometry/curve.jl

```diff
@@ -268,22 +304,31 @@ Given a divisor `dtc` and vertex labelled `position`, compute the linearly equiv
 julia> IM = IncidenceMatrix([[1,2],[1,3],[1,4],[2,3],[2,4],[3,4]]);
 
 julia> tc = TropicalCurve(IM)
-Abstract min tropical curve
+ERROR: MethodError: no method matching TropicalCurve(::Polymake.LibPolymake.IncidenceMatrixAllocated{Polymake.LibPolymake.NonSymmetric})
```

Comment:
Error here and below "intentional" (or rather: known and expected)?

Reply:
No, the erros are not known but also not expected.  Let me just quickly check with some people downstairs in Paderborn.

Reply:
Any progress here, or plans for progress, or things others could help with?

Reply:
Not yet. I am currently polishing the OSCAR chapter and this together, based on the reviews that I have received.